test-suite/src/test/scala/org/scalajs/testsuite/javalib/math/BigIntegerNotTest.scala

```diff
@@ -1,3 +1,4 @@
+// scalastyle:off
```

Comment:
This seams like something that we will forget there and that no one will ever fix this. It might be a good idea to put a some explicit tag like a `// TODO: fix style` to have clear reminder to do it eventually.